src/org/whstsa/library/gui/api/GuiNewLibrary.java

```diff
@@ -17,12 +17,13 @@ public Scene draw() {
         Button cancelButton = GuiUtils.createButton("Cancel", GuiUtils.defaultClickHandler());
         cancelButton.addEventHandler(ActionEvent.ACTION, event -> GuiUtils.defaultCloseOperation(event));
 
-        InputGroup newLibraryTextField = GuiUtils.createInputGroup("New Library Text Field", true);
+       // InputGroup newLibraryTextField = GuiUtils.createInputGroup("New Library Text Field", true);
```

Comment:
I would remove commented out code unless you would like to use it in the future. Cameron may have already replaced this code.